tests/test_common.py

```diff
@@ -0,0 +1,276 @@
+import os
+import subprocess
+import time
+import pytest
+import random
+import logging
+from concurrent.futures import ThreadPoolExecutor
+from hashlib import sha256
+from pathlib import Path
+from typing import Set, Tuple
+
+from app.common import (
+    OTAFileCacheControl,
+    SimpleTasksTracker,
+    copytree_identical,
+    file_sha256,
+    re_symlink_atomic,
+    read_from_file,
+    subprocess_call,
+    subprocess_check_output,
+    verify_file,
+    write_to_file_sync,
+)
+from tests.utils import compare_dir
+
+logger = logging.getLogger(__name__)
+
+_TEST_FILE_CONTENT = "123456789abcdefgh" * 3000
+_TEST_FILE_SHA256 = sha256(_TEST_FILE_CONTENT.encode()).hexdigest()
+_TEST_FILE_LENGTH = len(_TEST_FILE_CONTENT)
```

Comment:
Not to depend on contents.
```suggestion
_TEST_FILE_LENGTH = len(_TEST_FILE_CONTENT.encode())
```